src/seedu/addressbook/ui/Formatter.java

```diff
@@ -43,10 +44,23 @@ public String format(List<? extends ReadOnlyPerson> persons) {
         return format(asIndexedList(formattedPersons));
     }
 
+    /**
+     * Returns a concatenated version of the printable strings of each object.
+     */
+    public String getPrintableString(Printable... printables){
+        StringBuilder sb = new StringBuilder();
```

Comment:
Good job using the StringBuilder class, another alternative is the StringJoiner Class.

Reference: https://docs.oracle.com/javase/8/docs/api/java/util/StringJoiner.html

Reply:
Noted with thanks.